### ISSUE TITLE
No additional questions on Andriod device

### ISSUE THREAD

**Issue by someren** [Signalen/frontend#137]
**Describe the bug**
We have two webforms: one for 's-Hertogenbosch and the other one specifically for the Afvalstoffendienst (url: https://meldingen.afvalstoffendienst.nl/incident/beschrijf). **This problem currently only occurs at the Afvalstoffendienst webform**. When using an Andriod device the additional questions are skipped. Instead of getting additional questions the form goes directly to the "add phonenumber" page.

**To Reproduce**

1. Use an Andriod device like a Samsung phone or tablet
2. Open a browser, it can be reproduced with Firefox, Chrome and Samsung internet
3. Go to "meldingen.afvalstoffendienst.nl"
4. Choose a location
5. Type the text "ondergrondse container is vol"
6. Choose "Now"
7. Click on "Verder"
8. Instead of getting additional questions, like what happens when using the webform meldingen.s-hertogenbosch.nl, you skip to the phonenumber page.

**Expected behavior**
The additional questions page is shown after typing text regarding ondergrondse containers.

**Screenshots**
-

**Desktop (please complete the following information):**
I could not reproduce this problem on a desktop

**Smartphone (please complete the following information):**

- Device: Samsung A52, Samsung S10e and Samsung tablet (type unknown)
- OS: Andriod
- Browser Samsung internet, Chrome, Firefox

**Comment by MauritsR** [comment]
@someren Can you still reproduce this issue? I'm getting the additional questions (on desktop as well as mobile)

**Comment by someren** [comment]
@MauritsR Hi Maurits, I can reproduce the issue on our acceptance environment (https://afvalstoffendienst.signalen.dev/incident/beschrijf) but no longer on production. On production the additional questions are presented, I can select them and the question+answer is shown in the report. 

On acceptance the additional questions are still skipped and I go directly to the telephone number page. I reproduced this issue on the Samsung A52 in Firefox (88.1.3) and Chrome (90.0.4430.210). I used the text "Ondergrondse container is vol".
